src/briefcase/platforms/web/static.py

```diff
@@ -0,0 +1,371 @@
+            )
+
+            # Extract static resources from packaged wheels
+            for wheelfile in sorted(self.wheel_path(app).glob("*.whl")):
+                self.logger.info(f"  Processing {wheelfile.name}...")
+                with briefcase_css_path.open("a", encoding="utf-8") as css_file:
+                    self._process_wheel(wheelfile, css_file=css_file)
+
+        return {}
+
+
+class HTTPHandler(SimpleHTTPRequestHandler):
+    """Convert any HTTP request into a path request on the static content
+    folder."""
+
+    def translate_path(self, path):
+        return str(self.server.base_path / path[1:])
+
+
+class LocalHTTPServer(ThreadingHTTPServer):
+    """A HTTP server that serves local static content."""
+
+    def __init__(self, base_path, host, port, RequestHandlerClass=HTTPHandler):
+        self.base_path = base_path
+        super().__init__((host, port), RequestHandlerClass)
+
+
+class StaticWebRunCommand(StaticWebMixin, RunCommand):
+    description = "Run a static web project."
+
+    def add_options(self, parser):
+        super().add_options(parser)
+        parser.add_argument(
+            "--host",
+            default="localhost",
+            help="The host on which to run the server (default: localhost)",
+            required=False,
+        )
+        parser.add_argument(
+            "-p",
+            "--port",
+            default=8080,
+            type=int,
+            help="The port on which to run the server (default: 8080)",
+            required=False,
+        )
+        parser.add_argument(
+            "--no-browser",
+            action="store_false",
+            dest="open_browser",
+            help="Don't open a web browser on the newly opened server.",
+            required=False,
+        )
+
+    def run_app(self, app: AppConfig, host, port, open_browser, **kwargs):
+        """Start the application.
+
+        :param app: The config object for the app
+        :param host: The host on which to run the server
+        :param port: The port on which to run the server
+        :param open_browser: Should a browser be opened on the newly started
+            server.
+        """
+        self.logger.info(
+            f"Starting web server on http://{host}:{port}/",
+            prefix=app.app_name,
+        )
+
+        httpd = None
+        try:
+            # Create a local HTTP server
+            httpd = LocalHTTPServer(
+                self.project_path(app),
+                host=host,
+                port=port,
+            )
+
+            # If requested, open a brower tab on the newly opened server.
+            if open_browser:
+                webbrowser.open_new_tab(f"http://{host}:{port}")
+
+            self.logger.info(
+                "Web server log output (type CTRL-C to stop log)...",
+                prefix=app.app_name,
+            )
+            self.logger.info("=" * 75)
```

Comment:
In lieu of disabling browser caching, I was thinking it might be nice to add a line like this here for folks that aren't perhaps steeped in web dev:

```
Tip: holding SHIFT while clicking refresh will bypass browser caching
```

Reply:
If it's going to cause enough problems to include that message, it's worth including the disable-cache headers. I'll add them.

Reply:
:) It just kept coming to mind because I was constantly running in to unexpected caching. even if i shutdown the server, made code changes, and start everything back up again.